### PR TITLE
Added units of measurement

### PR DESCRIPTION
**Pull request by MaaikevS**
Related to Issue #134

### CODE REVIEW

File: instances/unitOfMeasurement/degree.jsonld [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/degree",
```
Comment (lzehl) [inline instances/unitOfMeasurement/degree.jsonld]: This one should be specified/changed:
+ arcdegree (synonyms: degree of arc, degree, arc degree;) VS
+ degreeCelcius (degree of temperature; as you have below)
+ degreeFahrenheit (degree of temperature; synonym: degree)
+ degreeRankine (degree of temperature; synonym: degree)

File: instances/unitOfMeasurement/microgramPerMilliliter.jsonld [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/microgramPerMilliliter",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": null,
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "microgram per milliliter",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "ug/ml"
```
Comment (lzehl) [inline instances/unitOfMeasurement/microgramPerMilliliter.jsonld]: this should be also including a true `µg/ml`

File: instances/unitOfMeasurement/micromolar.jsonld [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "Micromolar is a decimal fraction of molar concentration that describes a solution as micromole per one liter of a solution.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "micromolar",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "uM",
+    "10^-6 mol/L",
```
Comment (lzehl) [inline instances/unitOfMeasurement/micromolar.jsonld]: this I would delete.

File: instances/unitOfMeasurement/micromolar.jsonld [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  "definition": "Micromolar is a decimal fraction of molar concentration that describes a solution as micromole per one liter of a solution.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "micromolar",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "uM",
+    "10^-6 mol/L", 
+    "10^-3 mol/m^3"
```
Comment (lzehl) [inline instances/unitOfMeasurement/micromolar.jsonld]: this I would delete

File: instances/unitOfMeasurement/micromolar.jsonld [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/micromolar",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "Micromolar is a decimal fraction of molar concentration that describes a solution as micromole per one liter of a solution.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "micromolar",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "uM",
```
Comment (lzehl) [inline instances/unitOfMeasurement/micromolar.jsonld:15]: should include `µM`

File: instances/unitOfMeasurement/degreeCelsius.jsonld [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  },
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/degreeCelsius",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": null,
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "degree celsius",
+  "preferredOntologyIdentifier": null,
+  "synonym": null
```
Comment (lzehl) [inline instances/unitOfMeasurement/degreeCelsius.jsonld]: degree

File: instances/unitOfMeasurement/milligramPerKilogramBodyWeight.jsonld [hunk 7]
```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/milligramPerKilogramBodyWeight",
```
Comment (lzehl) [inline instances/unitOfMeasurement/milligramPerKilogramBodyWeight.jsonld:5]: this is not a unit. "milligram per kilogram" is the unit. That the kg is based on the body weight is context specific (case in which the unit is used).

File: instances/unitOfMeasurement/millimolar.jsonld [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "Millimolar is a decimal fraction of molar concentration that describes a solution as millimole per one liter of a solution.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "millimolar",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "mM",
+    "10^-3 mol/L",
```
Comment (lzehl) [inline instances/unitOfMeasurement/millimolar.jsonld:15]: I would delete this

File: instances/unitOfMeasurement/millimolar.jsonld [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+  "definition": "Millimolar is a decimal fraction of molar concentration that describes a solution as millimole per one liter of a solution.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "millimolar",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "mM",
+    "10^-3 mol/L", 
+    "mol/m^3"
```
Comment (lzehl) [inline instances/unitOfMeasurement/millimolar.jsonld:16]: I would delete this

File: instances/unitOfMeasurement/millimolar.jsonld (hunk 8 above)
Comment (lzehl) [reply instances/unitOfMeasurement/millimolar.jsonld:15]: this should be `mmol/l` or `mmol/L`

File: instances/unitOfMeasurement/millimolar.jsonld (hunk 9 above)
Comment (lzehl) [reply instances/unitOfMeasurement/millimolar.jsonld:16]: this should be `mmol*m⁻³` or `mmol/m³`

File: instances/unitOfMeasurement/degreeCelsius.jsonld (hunk 6 above)
Comment (lzehl) [reply instances/unitOfMeasurement/degreeCelsius.jsonld]: °C

File: instances/unitOfMeasurement/degree.jsonld [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": null,
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "degree",
+  "preferredOntologyIdentifier": null,
+  "synonym": null
```
Comment (lzehl) [inline instances/unitOfMeasurement/degree.jsonld]: `deg`
`°`

File: instances/unitOfMeasurement/micromolar.jsonld (hunk 3 above)
Comment (lzehl) [reply instances/unitOfMeasurement/micromolar.jsonld]: better `µmol/L`

File: instances/unitOfMeasurement/micromolar.jsonld (hunk 4 above)
Comment (lzehl) [reply instances/unitOfMeasurement/micromolar.jsonld]: better `µmol/m³` or `µmol*m⁻³`

File: instances/unitOfMeasurement/molar.jsonld [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+  "definition": "Molar is a measure of concentration that describes a solution as moles of solute per one liter of a solution.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "molar",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "M",
+    "mol/L", 
+    "10^3 mol/m^3"
```
Comment (lzehl) [inline instances/unitOfMeasurement/molar.jsonld:16]: this is wrong, should be `mol/m³` or `mol*m⁻³`

File: instances/unitOfMeasurement/molar.jsonld [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "Molar is a measure of concentration that describes a solution as moles of solute per one liter of a solution.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "molar",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "M",
+    "mol/L",
```
Comment (lzehl) [inline instances/unitOfMeasurement/molar.jsonld:15]: also `mol/l`

File: instances/unitOfMeasurement/nanomolar.jsonld [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "Nanomolar is a decimal fraction of molar concentration that describes a solution as nanomole per one liter of a solution.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "nanomolar",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "nM",
+    "10^-9 mol/L",
```
Comment (lzehl) [inline instances/unitOfMeasurement/nanomolar.jsonld]: see comment above

File: instances/unitOfMeasurement/nanomolar.jsonld [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+  "definition": "Nanomolar is a decimal fraction of molar concentration that describes a solution as nanomole per one liter of a solution.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "nanomolar",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "nM",
+    "10^-9 mol/L", 
+    "10^-6 mol/m^3"
```
Comment (lzehl) [inline instances/unitOfMeasurement/nanomolar.jsonld]: see comments above

File: instances/unitOfMeasurement/percent.jsonld [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+  },
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/percent",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": null,
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "percent",
+  "preferredOntologyIdentifier": null,
+  "synonym": null
```
Comment (lzehl) [inline instances/unitOfMeasurement/percent.jsonld]: `%`

Comment (lzehl) [reply instances/unitOfMeasurement/percent.jsonld]: `percentage`

Comment (lzehl) [reply instances/unitOfMeasurement/percent.jsonld]: `pct` and `pc`

File: instances/unitOfMeasurement/volumePerVolumePercentage.jsonld [hunk 16]
```diff
@@ -0,0 +1,16 @@
...
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/volumePerVolumePercentage",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "Percentage of volume of one liquid dissolved in another liquid",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "volume per volume percentage",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "% v/v"
```
Comment (lzehl) [inline instances/unitOfMeasurement/volumePerVolumePercentage.jsonld]:
```
percent v/v
v/v percent
v/v percent
volume percent
v/v %
```

File: instances/unitOfMeasurement/weightPerVolumePercentage.jsonld [hunk 17]
```diff
@@ -0,0 +1,16 @@
...
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/weightPerVolumePercentage",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "Percentage of weight of a dissolved, solid substance in a total volume of a solution",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "weight per volume percentage",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "% w/v"
```
Comment (lzehl) [inline instances/unitOfMeasurement/weightPerVolumePercentage.jsonld]: cf. comments above 
AND
w => should be g
v => should be ml
[as far as I saw]

File: instances/unitOfMeasurement/volumePerVolumePercentage.jsonld (hunk 16 above)
Comment (lzehl) [reply instances/unitOfMeasurement/volumePerVolumePercentage.jsonld]: volumes should have same unit

File: instances/unitOfMeasurement/weightPerWeightPercentage.jsonld [hunk 18]
```diff
@@ -0,0 +1,16 @@
...
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/weightPerWeightPercentage",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "The percentage of a particular substance within a mixture, as measured by weight or mass",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "weight per weight percentage",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "% w/w"
```
Comment (lzehl) [inline instances/unitOfMeasurement/weightPerWeightPercentage.jsonld]: cf. comments above
weight should have same unit

**Review by lzehl — CHANGES_REQUESTED** [review]

**Comment by MaaikevS** [comment]
@lzehl Your requested changes are incorporated

**Review by lzehl — APPROVED** [review]
@MaaikevS thanks a lot. I've only made some corrections for the definitions and synonyms.

**Pull request merged**